easybuild/tools/job/slurm.py

```diff
@@ -147,9 +149,6 @@ def __init__(self, script, name, env_vars=None, hours=None, cores=None):
 
         self.job_specs = {'wrap': self.script, 'job-name': self.name}
 
-        if env_vars:
-            self.job_specs['export'] = ','.join(sorted(env_vars.keys()))
```

Comment:
@akesandgren This should be fixed elsewhere imho... I.e. we should leave support for passing down particular environment variables in place, but simply not let `eb` pass down anything in general (rather than only for a particular job backend)